stdlib/typing_extensions.pyi

```diff
@@ -52,6 +52,9 @@ def IntVar(name: str) -> Any: ...  # returns a new TypeVar
 
 # Internal mypy fallback type for all typed dicts (does not exist at runtime)
 class _TypedDict(Mapping[str, object], metaclass=abc.ABCMeta):
+    __required_keys__: frozenset[str]
```

Comment:
`_TypedDict` from `typing.pyi` does not have these values as well: https://github.com/python/typeshed/blob/master/stdlib/typing.pyi#L681